.github/workflows/pull-validate-schema-migrator.yaml

```diff
@@ -0,0 +1,18 @@
+name: Validate database migrations
+
+on:
+  pull_request:
+    branches: [ main ]
+    types: [ opened, synchronize, reopened, ready_for_review ]
+    paths:
+      - 'resources/keb/migrations/**.sql'
+
```

Comment:
I think it should be expanded with paths that also include the source code of the schema migrator, so that if we change something in the app, we can test it.

Reply:
Also with the path to the workflow itself and the script that runs it.

Reply:
Good ideas, I will add path patterns.